shared_model/interfaces/query_responses/transactions_response.hpp

```diff
@@ -20,12 +20,12 @@
 
 #include <rxcpp/rx-observable.hpp>
```

Comment:
Remove this line as well plz, shouldn't be here

Reply:
done